src/Kunstmaan/TranslatorBundle/Command/ImportTranslationsCommand.php

```diff
@@ -11,7 +11,11 @@
 
 /**
  * @final since 5.1
+ * @deprecated since 6.0
  * NEXT_MAJOR extend from `Command` and remove `$this->getContainer` usages
+ * NEXT MAJOR file will be renamed
```

Comment:
Maybe use `NEXT_MAJOR` so it's the same as the line above, makes searching for occurrences easier